packages/jest-each/src/__tests__/__snapshots__/array.test.js.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`jest-each .describe throws an error when not called with an array 1`] = `
+"\`.each\` must be called with an Array or Tagged Template String.
```

Comment:
should we have a "was called with" thing?

Reply:
just added it :)